.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,30 @@
+# https://github.com/googleapis/repo-automation-bots/tree/main/packages/sync-repo-settings
+# Rules for main branch protection
+branchProtectionRules:
+# Identifies the protection rule pattern. Name of the branch to be protected.
+# Defaults to `main`
+- pattern: main
+  requiresCodeOwnerReviews: true
+  requiresStrictStatusChecks: true
+  requiredStatusCheckContexts:
+    - 'cla/google'
+    - 'OwlBot Post Processor'
+    - 'Kokoro'
+    - 'Kokoro SQLAlchemy compliance'
```

Comment:
I'm undecided as to whether we should block PRs for this. They can take hours to run and we don't have direct control over them, so I'm inclined to say they shouldn't be blocking (but if they're broken we should fast-follow with a fix)

Reply:
Side note: it'd be lovely if we could have this block release PRs only.